site/content/docs/5.1/components/title-bars.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: docs
+title: Title bars
+description: Documentation and examples for Boosted's exclusive Brand responsive title bars.
+group: components
+toc: true
+---
+
+## How it works
+
+A title bar is designed to display the title of a page and shall be placed right below a global header.
```

Comment:
Global header doesn't exist in Boosted doc. Maybe talk about (+ link) Orange navbar here.